views/index.html

```diff
@@ -3,6 +3,7 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="icon" type="image/x-icon" href="C:\Users\MSCA\Desktop\Backend lessons\projects\mysqltest\vigilant-kanye-tribble\assets\images\eco-green-earth-icon-24-182122.png">
```

Comment:
try with just `\assets\images\eco-green-earth-icon-24-182122.png` instead of the really long file name